### ISSUE TITLE
Update schema records on merge to main

### ISSUE THREAD

**Issue by ngerakines** [lexicon-community/lexicon#2]
When a PR with changes to `community/lexicon/**` is merged into the main branch, update the corresponding records in the `@lexicon.community` PDS.

Depends on https://github.com/lexicon-community/lexicon/issues/1.

**Comment by ebwinters** [comment]
@ngerakines Do you have any outline of how this can be done? Are you thinking of having some migration endpoint on the PDS that GH actions will call into and update all records? Maybe I am also misunderstanding the idea here. Curious to hear any rough plans around this and happy to help as well

**Comment by ngerakines** [comment]
A python script with env variables (GH action secrets) to adjust DNS records and then to make the "repository put" API call. Waiting on the RFC for lexicon resolution to move before starting on that.

**Comment by bnewbold** [comment]
I'm working on a `goat` sub-command which publishes individual lexicons to a repo. It wouldn't be hard to "sync" an entire directory of lexicon JSON files (recursively), in a way that can be used for CI/CD, if that would be helpful. Would need to find a way to distribute `goat`... docker? or maybe just invoking `go run` or `go get`.

**Comment by bnewbold** [comment]
Here is that `goat` tool, still in a PR, if you want to experiment with it or have feedback:
https://github.com/bluesky-social/indigo/pull/888

The "public" functionality is pretty minimal. In theory it could do something more complex like a "sync" command to recursively update from a directory of lexicon JSON files.

**Comment by ngerakines** [comment]
@bnewbold that is awesome! Looking at the code, the reverse NSID with `_lexicon.` maps to a plain DID (no prefix, suffix, or quotes). Sounds pretty straight forward!
